metrics_api/lib/opentelemetry/metrics/instrument/observable_up_down_counter.rb

```diff
@@ -9,7 +9,15 @@ module Metrics
     module Instrument
       # No-op implementation of ObservableUpDownCounter.
       class ObservableUpDownCounter
-        # TODO
+        # Observe the ObservableCounter with fixed timeout duartion.
+        # Everytime observe, the value should be sent to backend through exporter
```

Comment:
This doesn't match with how I thought the asynchronous instruments worked. I thought you'd call `observe` once, perhaps in an initializer, and then it would be invoked on every export cycle by default or on intervals based on the provided timeout. Kind of like a heartbeat? 

Is it more like `record`, where you have to call it every time you want a measurement?

Reply:
Yes, you can record data by calling observe; and yes, the observe will be invoke every export cycle, not based on time interval. I think export based on time interval, just use periodical reader with async instrument (e.g. record cpu utilization).

This comment maybe misleading; I will update it.